### PR TITLE
(fix): show actual does not effect step log

### LINKED ISSUE #2101: customLocator 'showActual' property does not affect step log
#### What are you trying to achieve?

To see the actual CSS selector that our customLocator produces, in the log, when we run tests with `--steps`. (E.g. `[data-test="my-value"]`)

This is what I understood setting the `showActual` config to true should do:

"`showActual` (default: false) show in the output actually produced XPath or CSS locator. By default shows custom locator value."

#### What do you get instead?

It just displays the custom locator value (e.g. `$my-value`).

I can see that the customLocator plugin is evaluating `showActual` to true and setting `locatorObj.output` to "[data-test....]". However, the step `toString` method does not log the output, only the step’s `args`.

> Provide console output if related. Use `--verbose` mode for more details.

Here's example log output:

```
    I: waitAndSee 
      I wait for element "$carrier-bag-charge-label"
      I see "Carrier bag charge:", "$carrier-bag-charge-label"
```

> Provide test source code if related

```js
I.waitAndSee('Carrier bag charge:', '$carrier-bag-charge-label');
```

### Details

* CodeceptJS version: v2.3.6
* NodeJS Version: v12.4.0
* Operating System: Mac OS 10.14

* puppeteer
* Configuration file:

Here's the relevant part from our config:

```js
    customLocator: {
      attribute: 'data-test',
      enabled: true,
      showActual: true,
      strategy: 'css',
    },
```

### PR DESCRIPTION
**Pull request by PeterNgTr**
## Motivation/Description of the PR
Fixes #2101 

**Before**
```sh
Test --
  test something
    I am on page ""
    I dont see "Carrier bag charge:", "$carrier-bag-charge-label"
  ✖ FAILED in 5836ms
```

**After**
```sh
Test --
  test something
    I am on page ""
    I dont see "Carrier bag charge:", ".//*[@data-qa='carrier-bag-charge-label']"
  ✖ FAILED in 5630ms
```

## Type of change

- [ ] Breaking changes
- [ ] New functionality
- [x] Bug fix
- [ ] Documentation changes/updates
- [ ] Hot fix
- [ ] Markdown files fix - not related to source code

### CODE REVIEW

**Comment by DavertMik** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- lib/output.js  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/DavertMik/CodeceptJS/pullRequest?prid=4841498&bid=16074729)

File: lib/output.js [hunk 1]
```diff
@@ -95,6 +95,12 @@ module.exports = {
    * @param {CodeceptJS.Step} step
    */
   step(step) {
+    const Locator = require('./locator');
```
Comment (Vorobeyko) [inline lib/output.js]: Can you write test on this changes?

File: lib/output.js [hunk 2]
```diff
@@ -95,6 +95,12 @@ module.exports = {
...
+    if (step.args[1]) {
+      const locator = new Locator(step.args[1]);
```
Comment (DavertMik) [inline lib/output.js]: What if 1st argument of a step **is not locator**

Comment (PeterNgTr) [reply lib/output.js]: I think I miss something here, cause the `step.args[0]` or `step.args[1]` could be the locator.

File: lib/output.js (hunk 1 above)
Comment (PeterNgTr) [reply lib/output.js]: will do :)

**Comment by DavertMik** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- test/data/sandbox/customLocator_test.js  1
- lib/output.js  2
         

Clones added
============
- test/data/sandbox/codecept.customLocator.showActualFalse.js  1
- test/data/sandbox/codecept.customLocator.showActualTrue.js  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/DavertMik/CodeceptJS/pullRequest?prid=4841498&bid=16074729)

**Comment by DavertMik** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- test/data/sandbox/customLocator_test.js  1
- lib/output.js  3
         

Clones added
============
- test/data/sandbox/codecept.customLocator.showActualFalse.js  1
- test/data/sandbox/codecept.customLocator.showActualTrue.js  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/DavertMik/CodeceptJS/pullRequest?prid=4841498&bid=16074729)

**Comment by DavertMik** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- test/data/sandbox/customLocator_test.js  1
- lib/output.js  5
         

Clones added
============
- test/data/sandbox/codecept.customLocator.showActualFalse.js  1
- test/data/sandbox/codecept.customLocator.showActualTrue.js  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/DavertMik/CodeceptJS/pullRequest?prid=4841498&bid=16074729)

**Comment by DavertMik** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- lib/output.js  5
- test/data/sandbox/customLocator_test.js  1
         

Clones added
============
- test/data/sandbox/codecept.customLocator.showActualFalse.js  1
- test/data/sandbox/codecept.customLocator.showActualTrue.js  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/DavertMik/CodeceptJS/pullRequest?prid=4841498&bid=16074729)

**Comment by DavertMik** [comment]
Sorry, not this fix...

**Comment by PeterNgTr** [comment]
okay @DavertMik then I'll close this PR, anyone has better solution just goes for it! 😺

**Comment by DavertMik** [comment]
Yeah, I hope so 😁🙏

**Pull request closed without merging**